src/status_im/utils/prices.cljs

```diff
@@ -13,19 +13,24 @@
 (def api-url "https://min-api.cryptocompare.com/data")
 (def status-identifier "extraParams=Status.im")
 
+(defn ->url-param-syms [syms]
+  (if (coll? syms)
```

Comment:
Do we expect it to be a collection or a string? If we have full control over this and does not require too many changes we should probably always pass the same type ( a collection), for simplicity.